contents/blog/new-pricing.md

```diff
@@ -0,0 +1,143 @@
+| **∞**              | $0.0000090 / event       | $0.0000100 / event                | -10%           |
+
+## When should I use the new option?
+
+We use it for events where we're only interested in aggregate data, such as for logged-out users on our website.
+
+It could also be useful when:
+
+- You're willing to lose some functionality for lower bills.
+- You want to track API or server events, especially at high volumes. 
+- You run a large e-commerce or content website with millions of users.
+- You're not tracking a high-volume event because it would be prohibitively expensive.
+- You want to use [autocapture](/docs/product-analytics/autocapture) to track events, but have turned it off to save money.
+
+## How do I start using it?
+
+For most people, it only requires a simple config change to include `process_persons: "identified_only"`. 
+
+This will track anonymous traffic without person profiles, and start collecting person information any time an identifying action is taken (i.e. using `identify()`, `group()`, setting person properties with `$set`, etc).
+
+You can get instructions for this and other use cases in our [persons documentation](/docs/data/persons).
+
+## What's the downside?
+
+When sending events without person information, you cannot:
+
+- Filter on persons (e.g. an individual user)
+- Create cohorts filtered on person properties (e.g. paying vs non-paying users)
+- Target by person properties for feature flags, A/B tests and surveys.
+- Receive initial UTM values for tracking marketing campaign performance (only the most recent).
+- View a person’s profile in the app, or query the persons table in our SQL insights.
+- Use [groups functionality](/docs/product-analytics/group-analytics) on standard events (without person profiles).
+
+If you need the above functionality, just continue to send person information with your events (no need to change anything), for events where the specific functionality is needed. 
+
+Person profiles are not an all-or-nothing thing. You can use them on any traffic where you need more detailed information, and skip them when you don't need that detail.
+
+## Why are we doing this?
+
+The short answer? Because we can.
+
+The slightly longer answer? Because it's the right thing to do for both our users and our business.
+
+Most of our competitors are inefficient. They employ huge outbound sales teams to grow revenue. Their salaries and commissions for closing deals are passed onto their customers through higher prices.
+
+In contrast, we're 100% inbound. We grow mostly through word of mouth, and we charge based on actual usage. 
+
+We don't believe in loss-leaders, so we make a modest positive margin on each event sold, but what we charge is directly connected to what it costs us, not what we think we get away with charging.
+
+We grow our revenue through helping you grow, and onboarding you onto other tools, like session replay, feature flags, and surveys as well.
```

Comment:
Should we link to pricing principles or some strategy bit of handbook here?

Reply:
Pricing principles need updating atm, so left that out for now.